CWE/events/italy.txt

```diff
@@ -482,8 +482,8 @@ country_event = {
 		
 		election = no
 		
-		#DC was the incumbent going into the election
-		OR = { ruling_party_ideology = conservative ruling_party_ideology = big_tent }
+		#DC or PCI was the incumbent going into the election
+		OR = { ruling_party_ideology = conservative ruling_party_ideology = big_tent ruling_party_ideology = communist }
```

Comment:
Sorry I can't allow this. As this is a historical event, the conditions must reflect the situation at the time the historical event fired which in this case was DC as the incumbent government.

Reply:
Compromise solution is to check for ideology percentages. If the percentages are similar to the historical result then the event can fire.